cob_bms_driver/src/fake_bms.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import rospy
+from std_msgs.msg import Bool
+from std_msgs.msg import Float64
+from cob_srvs.srv import SetFloat, SetFloatResponse
+from std_srvs.srv import SetBool, SetBoolResponse
+
+class FakeBMS(object):
+    def __init__(self):
+        self.srv_charging = rospy.Service('~set_charging', SetBool, self.charging_cb)
+        self.srv_relative_remaining_capacity = rospy.Service('~set_relative_remaining_capacity', SetFloat, self.relative_remaining_capacity_cb)
+        self.pub_voltage            = rospy.Publisher('/bms/voltage', Float64, queue_size = 1)
+        self.pub_current            = rospy.Publisher('/bms/current', Float64, queue_size = 1)
+        self.pub_remaining_capacity = rospy.Publisher('/bms/remaining_capacity', Float64, queue_size = 1)
+        self.pub_charge_capacity    = rospy.Publisher('/bms/full_charge_capacity', Float64, queue_size = 1)
+        self.pub_temparature        = rospy.Publisher('/bms/temperature', Float64, queue_size = 1)
+        self.pub_charging_state     = rospy.Publisher('/bms/battery_charging', Bool, queue_size = 1)        
+        
+        self.voltage            = 0.0
+        self.current            = 0.0
+        self.remaining_capacity = 0.0
+        self.charge_capacity    = 0.0
+        self.temperature        = 0.0
+        self.charging_state     = False
+
+        rospy.Timer(rospy.Duration(0.1), self.timer_cb)
```

Comment:
can we use same poll/publish frequency as in the real driver? (see parameter https://github.com/ipa320/cob_driver/blob/indigo_dev/cob_bms_driver/src/cob_bms_driver_node.cpp#L180)